src/ui/NoNameFacet/NoNameFacetValues/NoNameFacetValuesList.ts

```diff
@@ -0,0 +1,22 @@
+import { $$, Dom } from '../../../utils/Dom';
+import { NoNameFacet } from '../NoNameFacet';
+import { NoNameFacetValue } from './NoNameFacetValue';
+import { NoNameFacetValueElement } from './NoNameFacetValueElement';
+
+export class NoNameFacetValuesList {
+  private list: Dom;
+  public element: HTMLElement;
+
+  constructor(private facet: NoNameFacet) {
+    this.list = $$('ul', { className: 'coveo-facet-values' });
+    this.element = this.list.el;
+  }
+
+  public renderValues(facetValues: NoNameFacetValue[]) {
```

Comment:
I think this could be merged with `NoNameFacetValues` and this class for be removed.

Both of them are dealing with list of facet values, and it seems dangerous that they could get "unsynched", since we now have to deal with two different list.

As much as possible we should aim to have a single source of truth as to what the current list of facet values is, and what each of their state is (selected/deselected).